packages/xo-server/src/api/hub.js

```diff
@@ -0,0 +1,35 @@
+export async function downloadAndInstallResource({
```

Comment:
This can be exposed by `xo-server-cloud` directly with `addApiMethod`. Can be done in a future PR though.

Reply:
Hub feature is a part of XO, I don't think that a plugin is relevant for this.